test/interactions/keyInteractionTests.ts

```diff
@@ -4,91 +4,62 @@ var assert = chai.assert;
-
-      var keyInteraction = new Plottable.Interactions.Key();
-
-      var aCode = 65; // "a" key
-      var bCode = 66; // "b" key
-
-      var aCallbackCalled = false;
-      var aCallback = () => aCallbackCalled = true;
-      var bCallbackCalled = false;
-      var bCallback = () => bCallbackCalled = true;
-
-      keyInteraction.onKeyPress(aCode, aCallback);
-      keyInteraction.onKeyPress(bCode, bCallback);
-      keyInteraction.attachTo(component);
-
-      var $target = $(component.background().node());
-
-      TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-      $target.simulate("keydown", { keyCode: aCode });
-      assert.isTrue(aCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
-      assert.isFalse(bCallbackCalled, "callback for \"b\" was not called when \"a\" key was pressed");
-
-      aCallbackCalled = false;
-      $target.simulate("keydown", { keyCode: bCode });
-      assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when \"b\" key was pressed");
-      assert.isTrue(bCallbackCalled, "callback for \"b\" was called when \"b\" key was pressed");
-
-      TestMethods.triggerFakeMouseEvent("mouseout", component.background(), -100, -100);
+      $target = $(component.background().node());
       aCallbackCalled = false;
-      $target.simulate("keydown", { keyCode: aCode });
-      assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when not moused over the Component");
-
-      svg.remove();
+      bCallbackCalled = false;
     });
 
-    it("appropriate keyCode is sent to the callback", () => {
-      var svg = TestMethods.generateSVG(400, 400);
-      var component = new Plottable.Component();
-      component.renderTo(svg);
-
-      var keyInteraction = new Plottable.Interactions.Key();
-
-      var bCode = 66; // "b" key
-
-      var bCallbackCalled = false;
-      var bCallback = (keyCode: number) => {
-        bCallbackCalled = true;
-        assert.strictEqual(keyCode, bCode, "keyCode 65(a) was sent to the callback");
-      };
-
-      keyInteraction.onKeyPress(bCode, bCallback);
-
-      keyInteraction.attachTo(component);
-
-      var $target = $(component.background().node());
-
-      TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-      $target.simulate("keydown", { keyCode: bCode });
-      assert.isTrue(bCallbackCalled, "callback for \"b\" was called when \"b\" key was pressed");
-
+    afterEach(() => {
       svg.remove();
     });
 
-    it("canceling callbacks is possible", () => {
-        var svg = TestMethods.generateSVG(400, 400);
-        var component = new Plottable.Component();
-        component.renderTo(svg);
-
-        var keyInteraction = new Plottable.Interactions.Key();
-
-        var aCode = 65; // "a" key
-
-        var aCallbackCalled = false;
-        var aCallback = () => aCallbackCalled = true;
-
+    describe("onKeyPress", () => {
+      beforeEach(() => {
```

Comment:
I personally used to like to have a describe block for every method as then there is a nice grouping in interacting with that functionality, and so I tried to use this idea where I could.

After thinking about it again though, I think that might have been a mistake as the function could technically be used outside the block.